src/lib/es6.d.ts

```diff
@@ -230,22 +230,22 @@ interface ArrayLike<T> {
 
 interface Array<T> {
     /** Iterator */
-    [Symbol.iterator] (): Iterator<T>;
+    [Symbol.iterator](): IterableIterator<T>;

```

Comment:
Interesting.  It returns something htat is both an iterator, and something you can get an iterator off of?